installers/install.ps1

```diff
@@ -175,7 +175,7 @@ $PSDefaultParameterValues['*:Encoding'] = 'utf8'
 
 # Run the installer.
 $env:ACTIVESTATE_SESSION_TOKEN = $script:SESSION_TOKEN_VALUE
-& $exePath $args --source-installer="install.ps1"
+& $exePath $args --channel="$script:CHANNEL" --source-installer="install.ps1"
```

Comment:
I don't think there's any reason for us to pass this at all anymore. The script is what handles the channel flag. The installer just installs the payload it has alongside it, and has the channel baked into it.